fe/fe-core/src/main/java/com/starrocks/common/FeNameFormat.java

```diff
@@ -32,7 +32,7 @@ public class FeNameFormat {
     // Now we can not accept all characters because current design of delete save delete cond contains column name
     // so it can not distinguish whether it is an operator or a column name
     // the future new design will improve this problem and open this limitation
-    private static final String COLUMN_NAME_REGEX = "^[^\0=<>!\\*]{1,64}$";
+    private static final String COLUMN_NAME_REGEX = "^[^\0=<>!\\*]{1,1023}$";
```

Comment:
1024?

Reply:
I'm not sure this limitation can actually work. Maybe somewhere there's a limit that only occurs in corner cases. I'm setting this value to 1023 because it's safer.